fastlane/spec/private_public_fastfile_spec.rb

```diff
@@ -25,6 +25,7 @@
       end
 
       it "doesn't expose the private lanes in `fastlane lanes`" do
+        require 'fastlane/lane_list'
```

Comment:
I thought if someone adds specs, they face up to the same situation.

However, according to other specs, there is no rule.  So either is fine. 👍